src/web/web-service/wpan_service.cpp

```diff
@@ -292,7 +292,7 @@ std::string WpanService::HandleStatusRequest()
         networkInfo["IPv6:MeshLocalPrefix"] = rval;
 
         meshLocalPrefix = rval;
-        meshLocalPrefix.resize(meshLocalPrefix.find('/'));
+        meshLocalPrefix.resize(meshLocalPrefix.find(":/"));
```

Comment:
Could you provide the `meshLocalPrefix` value causes this issue? I don't see there are cases that multiple `/` exist in this value.

Reply:
Hi, bukepo,
Commit ID: 089271cbf530239072fa426af9896efd21019b73
I use the OT Border Router Web to form the thread network with default params and get success.
![image](https://user-images.githubusercontent.com/20263334/100708101-b2314480-33e6-11eb-8d7b-73a3f2a3a82b.png)

```
pi@raspberrypi:~ $ sudo ot-ctl dataset
Active Timestamp: 1
Channel: 15
Channel Mask: 0x07fff800
Ext PAN ID: 1111111122222222
Mesh Local Prefix: fdc1:28ef:f75f:68e::/64
Master Key: 00112233445566778899aabbccddeeff
Network Name: OpenThreadDemo
PAN ID: 0x1234
PSKc: 2ea1e4c98e7d09b159315164d09dd27d
Security Policy: 0, onrcb
Done
```
when I refresh to view the Status page I get nothing:
![image](https://user-images.githubusercontent.com/20263334/100708357-19e78f80-33e7-11eb-8c5c-e423123f12d3.png)

with my fix and build again and after reboot:

```
pi@raspberrypi:~ $ sudo ot-ctl state
leader
Done
pi@raspberrypi:~ $ sudo ot-ctl dataset
Done
pi@raspberrypi:~ $ sudo ot-ctl dataset active
Active Timestamp: 0
Channel: 15
Channel Mask: 0x07fff800
Ext PAN ID: 1111111122222222
Mesh Local Prefix: fdc1:28ef:f75f:68e::/64
Master Key: 00112233445566778899aabbccddeeff
Network Name: OpenThreadDemo
PAN ID: 0x1234
PSKc: 61e1206d2c2b46e079eb775f41fc7219
Security Policy: 672, onrcb
Done
```

the Status page shows normal.
![image](https://user-images.githubusercontent.com/20263334/100710025-0b4ea780-33ea-11eb-8b7b-3575c2e49e8a.png)

Reply:
```
pi@raspberrypi:~ $ sudo ot-ctl dataset active
Active Timestamp: 0
Channel: 15
Channel Mask: 0x07fff800
Ext PAN ID: 1111111122222222
Mesh Local Prefix: fd85:3045:9e57:9c31::/64
Master Key: 00112233445566778899aabbccddeeff
Network Name: OpenThreadDemo
PAN ID: 0x1234
PSKc: 61e1206d2c2b46e079eb775f41fc7219
Security Policy: 672, onrcb
Done
pi@raspberrypi:~ $ ifconfig wpan0
wpan0: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST>  mtu 1280
        inet6 fd85:3045:9e57:9c31:0:ff:fe00:ec00  prefixlen 64  scopeid 0x0<global>
        inet6 fd85:3045:9e57:9c31:141:30f6:546c:4848  prefixlen 64  scopeid 0x0<global>
        inet6 fe80::50d6:19bb:1e86:7e75  prefixlen 64  scopeid 0x20<link>
        inet6 fd85:3045:9e57:9c31:0:ff:fe00:fc00  prefixlen 64  scopeid 0x0<global>
        unspec 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  txqueuelen 500  (UNSPEC)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 5  bytes 920 (920.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

```

if use "/"
https://github.com/openthread/ot-br-posix/blob/c8c0e81da69c99d689ed3145461ea598fba3f591/src/web/web-service/wpan_service.cpp#L295 
then
![image](https://user-images.githubusercontent.com/20263334/100720979-b9158280-33f9-11eb-891a-95305d31b60a.png)


https://github.com/openthread/ot-br-posix/blob/c8c0e81da69c99d689ed3145461ea598fba3f591/src/web/web-service/wpan_service.cpp#L303 
Could not find "meshLocalPrefix" in "rval" because there is an extra colon.
So we should use ":/" as delim.